about.css

```diff
@@ -0,0 +1,472 @@
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: lato, sans-serif;
+}
+
+li {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+  text-transform: capitalize;
+}
+
+section h2 {
+  text-align: center;
+  margin: 1rem 0;
+  position: relative;
+}
+
+.upper {
+  display: none;
+}
+
+header {
+  background-image: url(images/bgtest.jpg);
+  background-repeat: no-repeat;
+  background-size: cover;
+  padding: 1rem;
+  padding-bottom: 4rem;
+}
+
+.logo {
+  display: none;
+}
+
+.icon i {
+  font-size: 30px;
+  color: #272a32;
+}
+
+nav ul {
+  position: absolute;
+  height: 100vh;
+  width: 60%;
+  background-color: rgb(235, 230, 230);
+  top: 0;
+  right: -100%;
+  padding: 2rem 1rem;
+  display: none;
+  transition: 0.5s linear;
+  z-index: 99;
+}
+
+.icon .fa-xmark {
+  display: none;
+}
+
+nav li {
+  margin-bottom: 2rem;
+}
+
+nav a {
+  font-size: 23px;
+  color: rgb(58, 58, 58);
+  transition: 0.3s linear;
+}
+
+.intro a {
+  text-decoration: underline;
+  padding: 10px;
+}
+
+.links {
+  text-transform: capitalize;
+}
+
+footer p {
+  font-size: 14px;
+  margin-left: 1rem;
+  color: #272a32;
+}
+
+.intro p,
+.intro a {
+  color: #272a32;
+  font-size: 15px;
+}
+
+nav a:hover {
+  color: #ec5242;
+}
+
+footer h3 {
+  border: 2px solid #ec5242;
+  text-align: center;
+  padding: 1px 1rem;
+  font-size: 15px;
+  margin-right: 1rem;
+  color: #272a32;
+}
+
+.intro {
+  padding: 1.5rem 0;
+  text-transform: capitalize;
+  text-align: center;
+}
+
+.intro h3 {
+  font-size: 20px;
+  color: #ec5242;
+  font-weight: 400;
+}
+
+.intro h2 {
+  font-size: 25px;
+  margin-top: 15px;
+  color: #272a32;
+}
+
+.intro h1 {
+  color: #ec5242;
+  font-size: 32px;
+  line-height: 1.1;
+  margin: 0.5rem 0;
+  text-transform: uppercase;
+}
+
+.intro p {
+  margin: 0.5rem 0;
+  color: #272a32;
+}
+
+.intro p.bg {
+  background-color: rgb(245 243 243 / 93%);
+  color: #272a32;
+  border: 2px solid white;
+  padding: 10px;
+  margin: 1rem 0;
+}
+
+.show {
+  right: 0;
+  display: block;
+}
+
+section h2::after {
+  content: '';
+  width: 35px;
+  height: 2px;
+  border-radius: 12px;
+  position: absolute;
+  right: calc(50% - 8px);
+  bottom: -8px;
+  background-color: #ec5242;
+}
+
+.how .fa-bars {
+  display: none;
+}
+
+.how .fa-xmark {
+  display: block;
+}
+
+#larger {
+  display: none;
+}
+
+.ntslogo {
+  padding: 1rem;
+  text-align: center;
+  border-bottom: 1px solid rgb(206, 202, 202);
+  margin: 1rem 0;
+}
+
+.ntslogo h2 {
+  text-transform: capitalize;
+}
+
+.ntsimage {
+  width: 100%;
+  padding: 0 1rem;
+  margin: 1rem 0;
+  border: 1px solid rgb(206, 202, 202);
+}
+
+.ntsimage img {
+  width: 100%;
+}
+
```

Comment:
- [ ] Kindly resize the image of the logo so that your design looks identical to that of the template provided. Currently, the image size is too large. I would advise that you resize it.

### Your Design

![image](https://user-images.githubusercontent.com/77106962/210669043-5b92b465-e63e-4b46-b927-8df5d6a1216a.png)

### Template Design

![image](https://user-images.githubusercontent.com/77106962/210669130-83cbbd20-e098-41e8-9d64-b0729c74197b.png)